pkg/project/controller/controller.go

```diff
@@ -36,12 +35,6 @@ func (c *NamespaceController) Handle(namespace *kapi.Namespace) (err error) {
 		return nil
 	}
 
-	// there may still be content for us to remove
-	err = deleteAllContent(c.Client, namespace.Name)
-	if err != nil {
-		return err
-	}
-
 	// we have removed content, so mark it finalized by us
 	_, err = projectutil.Finalize(c.KubeClient, namespace)
```

Comment:
waiting for kube finalization should not be necessary since the kube finalizer will only remove itself when the origin grouped resources are deleted.